ledger/ledger-api-bench-tool/src/test/suite/scala/com/daml/ledger/api/benchtool/submission/CommandSubmitterITSpec.scala

```diff
@@ -166,6 +167,7 @@ class CommandSubmitterITSpec
     extends AsyncFlatSpec
     with SandboxFixture
     with SuiteResourceManagementAroundAll
+    with SandboxBackend.Postgresql
```

Comment:
This change looks independent of the new test config, is it intended?

Reply:
I wanted to give an example where it can be used and did a manual test of this new config using this spec.

Do you think it should go to a separate PR?

Reply:
I would recommend opening such PRs from main directly: 
* This needs to be reviewed twice anyway, since this merges back to your feature branch
* I'll have comments on this test on the bench-tool-nonconsuming feature branch
* I find it confusing to make an argument of a change, which depends on a change which is being reviewed
* This is anyway a tooling aid, which for your convenience you can have locally as well until it is merged

Reply:
Ok, removed this dependency and changed base to main